test/e2e/utils/test_context.go

```diff
@@ -243,10 +243,18 @@ func (t *TestContext) Destroy() {
 	//nolint:gosec
 	// if image name is not present or not provided skip execution of docker command
 	if t.ImageName != "" {
-		cmd := exec.Command("docker", "rmi", "-f", t.ImageName)
-		if _, err := t.Run(cmd); err != nil {
-			warnError(err)
+		// Remove white space from image name
+		trimmedImageName := strings.Trim(t.ImageName, "")
+		// Check trimmedimage name is empty
+		if len(trimmedImageName) < 0 {
```

Comment:
```suggestion
		if len(strings.Trim(t.ImageName, "")) < 0 {
```

You can do in the same line and you do not need spent a memory space to store the data in a variable

Otherwise it is great !!!